### DISCUSSION TITLE
guestbook

### DISCUSSION THREAD

**Discussion by giscus** [laymonage/isle#32] [Comments]
# guestbook

laymonage's guestbook. Feel free to sign it!

https://laymonage.com/guestbook

**Comment by laymonage** [comment]
Hello! :tada:

Leave a message :smile:

**Comment by thisstillwill** [comment]
Awesome project! Definitely going to try using it with my own site when I get a chance 👍

**Comment by daystram** [comment]
Sup! :rocket:

**Comment by Hans5958** [comment]
Came from giscus and that welcoming event. Keep up the good word, kak!

**Reply by devxan** [reply]
Hello there 👀

**Comment by qinyanjuidavid** [comment]
> fantastic

**Comment by harsxv** [comment]
This is great!

**Comment by thibaudcolas** [comment]
I want to have a Giscus guestbook on my site too!

**Reply by biplobsd** [reply]
My site too... 😂

**Comment by piharpi** [comment]
I found cool stuff here 🎉

**Comment by Stormheg** [comment]
Hi Sage!

**Comment by home-gihub** [comment]
amazing :+1:

**Comment by Tisen-Ray** [comment]
amazing!~\(≧▽≦)/~

**Comment by fahri-r** [comment]
keren mas!

**Comment by jsandburg** [comment]
Just leaving a quick little message to say that I came here to check out Giscus! I want comments on my site, but I'm using a static site generator for a blog this time around so I've been looking into how to best implement that. Thank you for your work on it.
